x-pack/test_serverless/api_integration/test_suites/observability/slos/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ loadTestFile }: FtrProviderContext) {
+  describe('SLOs', function () {
+    loadTestFile(require.resolve('./create_slo'));
+    loadTestFile(require.resolve('./delete_slo'));
+    // TODO delete them for now
```

Comment:
TODO 

Delete from this PR. Once stateful SLO integration tests are added as part of this [PR](https://github.com/elastic/kibana/pull/173236), reconsider how to approach adding more scenarios for serverless